pdf/footer.tpl

```diff
@@ -34,11 +34,11 @@
 			{if !empty($shop_phone) OR !empty($shop_fax)}
 				{l s='For more assistance, contact Support:' pdf='true'}<br />
 				{if !empty($shop_phone)}
-					Tel: {$shop_phone|escape:'html':'UTF-8'}
+					{l s='Tel:' pdf='true'} {$shop_phone|escape:'html':'UTF-8'}
```

Comment:
Hi,

Thank you for your contribution, what about this : ```{l s='Tel: %s' sprintf=[$shop_phone|escape:'html':'UTF-8'] pdf='true'}``` ? Like this you can preserve RTL translations.

Best regards.